src/main/java/org/jenkinsci/plugins/buildtriggerbadge/IconFinder.java

```diff
@@ -67,48 +62,40 @@ private String internalFindForClass(Class clazz)
 
 	private static final Map<String, String> DEFAULT_ICONS = new HashMap<String, String>();
 	static {
-
-		IMAGES_PATH = "/plugin/" + BuildTriggerBadgePlugin.get().getShortName() + "/images/";
-
-		defineIconForCause(UserIdCause.class, "user-cause.png");
-		defineIconForCause(UserCause.class, "user-cause.png");
-		defineIconForCause(TimerTriggerCause.class, "timer-cause.png");
-		defineIconForCause(SCMTriggerCause.class, "scm-cause.png");
-		defineIconForCause(UpstreamCause.class, "upstream-cause.png");
-		defineIconForCause(CLICause.class, "cli-cause.png");
-		defineIconForCause(RemoteCause.class, "remote-cause.png");
-		defineIconForCause("org.jvnet.hudson.plugins.m2release.ReleaseCause", "user-cause.png");
-		defineIconForCause("com.cloudbees.jenkins.GitHubPushCause", "github-push-cause.png");
-		defineIconForCause("org.jenkinsci.plugins.ghprb.GhprbCause", "github-pull-request-cause.png");
-		defineIconForCause("org.jenkinsci.plugins.github.pullrequest.GitHubPRCause", "github-pull-request-cause.png");
-		defineIconForCause("com.cloudbees.jenkins.plugins.github_pull.GitHubPullRequestCause", "github-pull-request-cause.png");
-		defineIconForCause("stashpullrequestbuilder.stashpullrequestbuilder.StashCause", "github-pull-request-cause.png");
-		defineIconForCause("io.jenkins.plugins.gitlabbranchsource.GitLabMergeRequestCommentCause", "gitlab-merge-request-comment.png");
-		defineIconForCause("org.jenkinsci.plugins.periodicreincarnation.PeriodicReincarnationBuildCause", "periodic-reincarnation.png");
-		defineIconForCause("com.chikli.hudson.plugin.naginator.NaginatorCause", "periodic-reincarnation.png");
-		defineIconForCause("com.cloudbees.plugins.flow.FlowCause", "flow-cause.png");
-		defineIconForCause("com.cloudbees.jenkins.plugins.BitBucketPushCause", "bitbucket.png");
-		defineIconForCause("hudson.plugins.git.GitStatus$CommitHookCause", "git-hook-cause.png");
-		defineIconForCause("org.jenkinsci.plugins.urltrigger.URLTriggerCause", "url-trigger-cause.png");
-		defineIconForCause("org.jenkinsci.plugins.gwt.GenericCause", "url-trigger-cause.png");
-		defineIconForCause("jenkins.branch.BranchIndexingCause", "branch-indexing-cause.png");
-		defineIconForCause("jenkins.branch.BranchEventCause", "branch-event-cause.png");
-		defineIconForCause("org.jenkinsci.plugins.workflow.cps.replay.ReplayCause", "periodic-reincarnation.png");
-		defineIconForCause("org.jenkinsci.lib.xtrigger.XTriggerCause", "xtrigger.png");
-		defineIconForCause("com.cloudbees.workflow.cps.checkpoint.RestoreFromCheckpointCause", "checkpoint.png");
-		defineIconForCause("org.jenkinsci.plugins.parameterizedscheduler.ParameterizedTimerTriggerCause", "timer-cause.png");
+		defineIconForCause(UserIdCause.class, "symbol-git plugin-buildtriggerbadge");
```

Comment:
is this bad copy/pase?  should be the same as User and not be related to `git` ?